tt_metal/hostdevcommon/common_runtime_address_map.h

```diff
@@ -90,38 +90,38 @@ constexpr static std::uint32_t OP_INFO_SIZE      = 280; // So far, holds up to 1
 constexpr static std::uint32_t DISPATCH_MESSAGE_ADDR = 110816;
 constexpr static std::uint64_t DISPATCH_MESSAGE_REMOTE_SENDER_ADDR = 110848;
 
-constexpr static std::uint32_t COMMAND_PTR_SHARD_IDX = 6;
+constexpr static std::uint32_t COMMAND_PTR_SHARD_IDX = 8;
```

Comment:
was this a bug?

Reply:
No, I added src and dst page offsets to support multiple read/write commands for one buffer